### PR TITLE
Add implement for Elena's greeting

### PR DESCRIPTION
**Pull request by lam00071**
Add new 'builder' and method for Elena's greeting

### CODE REVIEW

**Review by NicMcPhee — DISMISSED** [review]
Very nice – I like the use of Vietnamese!

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String elenaSaysHello() {
+    return "Elena says 'Xin chào!'\n";
```
Comment (biruk741) [inline src/main/java/hellos/Hellos.java:113]: I think you make some valid points here. Nice argument.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -19,6 +19,7 @@ public void run() {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(elenaSaysHello());
```
Comment (biruk741) [inline src/main/java/hellos/Hellos.java:25]: I like how you said append. Very cool.

**Review by biruk741 — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java:113]: I still think it's cool that we have a little Vietnamese in the project. 😎

**Review by NicMcPhee — DISMISSED** [review]
Looks good!

**Review by NicMcPhee — DISMISSED** [review]
Looks great!

**Review by biruk741 — DISMISSED** [review]

**Review by halleypaulson — DISMISSED** [review]

**Review by halleypaulson — APPROVED** [review]

**Pull request merged**